### PR TITLE
WD-10186 - feat: add user form to group panel

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Add a form to add users to a group

## QA

- Open the create group form.
- Click Add users
- Fill out the form
- Go back to the first panel and submit
- Use the FGA CLI to check that the user was added to the group

## Details

https://warthogs.atlassian.net/browse/WD-10186

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -1250,9 +1250,9 @@ components:
     IdentitiesPatchRequest:
       type: object
       required:
-        - permissions
+        - identities
       properties:
-        permissions:
+        identities:
```
Comment (huwshimi) [inline openapi.yaml:1255]: Discovered this param was misnamed.

File: src/components/pages/Groups/AddGroupPanel/AddGroupPanel.test.tsx [hunk 2]
```diff
@@ -67,3 +77,258 @@ test("should handle errors when adding groupss", async () => {
     "negative",
   );
 });
+
+test("should add entitlements", async () => {
```
Comment (huwshimi) [inline src/components/pages/Groups/AddGroupPanel/AddGroupPanel.test.tsx:81]: There were a couple of tests I'd missed here.

**Comment by huwshimi** [comment]
Needs: https://github.com/canonical/identity-platform-admin-ui/issues/283.

**Review by vladimir-cucu — APPROVED** [review]
Code and layout looks good to me. I also get the same 500 error for PATCH `*/groups/new-group-1/identities` with message:
```
"Write validation error for POST Write with body {\"code\":\"validation_error\",\"message\":\"Invalid tuple 'group:new-group-1#member#member@user:new-user-1'. Reason: invalid 'object' field format\"}\n with error code validation_error error message: Invalid tuple 'group:new-group-1#member#member@user:new-user-1'. Reason: invalid 'object' field format"
```

**Comment by huwshimi** [comment]
> Code and layout looks good to me. I also get the same 500 error for PATCH `*/groups/new-group-1/identities` with message:
> 
> ```
> "Write validation error for POST Write with body {\"code\":\"validation_error\",\"message\":\"Invalid tuple 'group:new-group-1#member#member@user:new-user-1'. Reason: invalid 'object' field format\"}\n with error code validation_error error message: Invalid tuple 'group:new-group-1#member#member@user:new-user-1'. Reason: invalid 'object' field format"
> ```

This has now been fixed on the API side. If you update your API and retry this branch it should now work.

**Review by vladimir-cucu — APPROVED** [review]
QAd against the API with latest changes and it works as expected. The following entries were created for the new group `new-group-1` with 1 user `new-user-1` and 0 entitlements:
```
  {
    "object":"group:new-group-1",
    "relation":"member",
    "user":"user:shipperizer"
  },
  {
    "object":"group:new-group-1",
    "relation":"privileged",
    "user":"privileged:superuser"
  },
  {
    "object":"group:new-group-1",
    "relation":"member",
    "user":"user:new-user-1"
  }
```

**Pull request merged**